src/MassTransit.MessagePack/Serialization/MessagePackMessageBody.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace MassTransit.Serialization;
+
+using System;
+using System.IO;
+using MessagePack;
+
+
+public class MessagePackMessageBody<TMessage> :
+    MessageBody
+    where TMessage : class
+{
+    public long? Length => _lazyMessagePackSerializedObject.Value.Length;
+
+    readonly Lazy<byte[]> _lazyMessagePackSerializedObject;
+
+    public MessagePackMessageBody(SendContext<TMessage> context, MessagePackEnvelope? envelope = null)
```

Comment:
Can nullable be enabled for the entire MessagePack project? Since it's new and all.

Reply:
Great suggestion! I have enabled `nullable` for the `MassTransit.MessagePack` project, and added nullable variable annotations where possible.